cmd/configschema/go.mod

```diff
@@ -2,7 +2,9 @@
 // See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30187
 module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/configschema
 
-go 1.21
+go 1.21.3
```

Comment:
is the specific version needed here?

Reply:
No my ide is being weird

Reply:
NVM it is a requirement of the package: https://github.com/nginxinc/nginx-prometheus-exporter/blob/main/go.mod. Soooo probably cant take it for another 6 months?